CODEOWNERS

```diff
@@ -12,5 +12,7 @@
 /Makefile @wilfriedroset @vaxvms @bubu11e @grafana/mimir-maintainers
 /packaging/ @wilfriedroset @vaxvms @bubu11e @grafana/mimir-maintainers
 
+# Security Workflows
+.github/workflows/sbom-report.yml @grafana/security-engineering
```

Comment:
Github complains about unknown owner here.

Reply:
We also probably want `@grafana/mimir-maintainers` here.

Reply:
Sounds good, let me know if any changes are needed 👍

Reply:
> Sounds good, let me know if any changes are needed

@SadFaceSmith could you please remove this change to `CODEOWNERS`?

Reply:
@56quarters Removed the `CODEOWNER` lines. 

@aknuds1 No, I'm not sure. I checked to make sure it's a real team in the `grafana` Org,